source/_components/calendar.todoist.markdown

```diff
@@ -0,0 +1,136 @@
+---
+layout: page
+title: "Todoist"
+description: "Instructions how to integrate Todoist into Home Assistant."
+date: 2017-08-31 2:22
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: todoist.png
+ha_category: Calendar
+ha_iot_class: "Cloud Polling"
+ha_release: 0.53
+---
+
+
+This platform allows you to connect to your [Todoist Projects](https://todoist.com) and generate binary sensors. A different sensor will be created for each individual project, or you can specify "custom" projects which match against criteria you set (more on that below). These sensors will be `on` if you have a task due in that project or `off` if all the tasks in the project are completed or if the project doesn't have any tasks at all. All tasks get updated roughly every 15 minutes.
+
+### {% linkable_title Prerequisites %}
+
+You need to determine your Todoist API token. This is fairly simple to do; just go [to the Integrations section on your Todoist settings page](https://todoist.com/Users/viewPrefs?page=authorizations) and find the section labeled "API token" at the bottom of the page. Copy that token and use it in your configuration file.
+
+### {% linkable_title Basic Setup %}
+
+To integrate Todoist in Home Assistant, add the following section to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+calendar:
+  - platform: todoist
+    token: API_token_created_from_steps_above
+```
+
+Configuration variables:
+
+- **token** (*Required*): The API token used to authorize Home Assistant to access your projects.
+- **custom_projects** (*Optional*): Details on any "custom" binary sensor projects you want to create.
+  - **name** (*Required*): The name of your custom project. Only required if you specify that you want to create a custom project.
+  - **due_date_days** (*Optional*): Only include tasks due within this many days. If you don't have any tasks with a due date set, this returns nothing.
+  - **labels** (*Optional*): Any labels you want to filter for. All tasks in this sensor will include this label.
```

Comment:
**Spelling & Grammar**:
- `for` needs to go. -> `Any labels you want to filter.` 
Alternatively, you could replace the whole description with: `Only include tasks which have this label`

**Unclear documentation**:
- If I set two labels, will the tasks actually need to have both? (is it AND or OR?)